css/stanford_news_grid.css

```diff
@@ -7,12 +8,257 @@
+.view-stanford-news-grid.views-grid-two .views-row.views-row-414,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-416,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-418,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-420,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-422,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-424,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-426,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-428,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-430,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-432,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-434,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-436,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-438,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-440,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-442,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-444,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-446,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-448,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-450,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-452,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-454,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-456,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-458,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-460,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-462,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-464,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-466,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-468,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-470,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-472,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-474,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-476,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-478,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-480,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-482,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-484,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-486,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-488,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-490,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-492,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-494,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-496,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-498,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-500 {
```

Comment:
Yes, a metric tonne

And Joe says that the .views-row:nth-child(2n) won't work if the view is grouped. [☹]

________________________________
From: Sherakama <notifications@github.com>
Sent: Monday, September 12, 2016 10:47:12 PM
To: SU-SWS/stanford_news
Cc: Caryl J Westerberg; Assign
Subject: Re: [SU-SWS/stanford_news] PEEC-16: added grid view (#38)


In css/stanford_news_grid.css<https://github.com/SU-SWS/stanford_news/pull/38#discussion_r78499556>:

> +.view-stanford-news-grid.views-grid-two .views-row.views-row-470,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-472,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-474,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-476,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-478,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-480,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-482,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-484,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-486,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-488,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-490,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-492,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-494,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-496,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-498,
> +.view-stanford-news-grid.views-grid-two .views-row.views-row-500 {


Oh gosh that is a tonne of statements.
I would like to introduce you to a wonderful shortcut for the 250 lines above.
view-stanford-news-grid.views-grid-two .views-row:nth-child(2n)

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/SU-SWS/stanford_news/pull/38/files/10a45921e002b0c94f108f0a31ebabaf879ccba3..705df77b0c8dbc0b862d1b28bc76f4d62436dc98#r78499556>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AARXGDc1HrHYDsEiKP629ZRAc88M9Fqfks5qpjjggaJpZM4JrW1N>.